catalogue/models/component/__init__.py

```diff
@@ -147,6 +147,61 @@ def __str__(self):
         return ""
 
 
+class Contact(models.Model):
+    # Contact Information
+    contact_person_surname = models.CharField(
+        "Vorname",
+        help_text="Vorname des zuständigen Mitarbeiters",
+        max_length=1000,
+    )
+    contact_person_lastname = models.CharField(
+        "Nachname",
+        help_text="Nachname des zuständigen Mitarbeiters",
+        max_length=1000,
+    )
+    contact_address_land = models.CharField(
+        "Land",
+        help_text="Land in welchem sich der Anbieter befindet",
+        max_length=1000,
+        default="Deutschland",
+    )
+    contact_address_zip = models.CharField(
+        "PLZ",
+        help_text="PLZ in der sich der Anbieter befindet",
+        max_length=1000,
+        blank=True,
+    )
+    contact_address_city = models.CharField(
+        "Stadt",
+        help_text="Stadt in der sich der Anbieter befindet",
+        max_length=1000,
+        blank=True,
+    )
+    contact_address_street = models.CharField(
+        "Straße",
+        help_text="Straße des Anbieters",
+        max_length=1000,
+        blank=True,
+    )
+    contact_email = models.CharField(
+        "Email",
+        help_text="Email Adresse des zuständigen Mitarbeiters",
+        max_length=1000,
+        blank=True,
+    )
+    contact_phone = models.CharField(
```

Comment:
Phone should have some format restrictions in place. Like start with `+`. We need the international phone format.

https://stackoverflow.com/a/63525095